src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface OperationHandler {
+    List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> list,
```

Comment:
The method name 'getFruitFinalQuantity' could be misleading in the context of an 'OperationHandler'. The method in an operation handler is usually expected to perform an operation such as 'handle' or 'execute'. Consider renaming the method to better reflect its purpose.